docs/src/main/paradox/bootstrap/recipes.md

```diff
@@ -54,11 +54,21 @@ Note that the `app` is the same for both services as they both refer to the same
 
 ### Health checks
 
-TODO make health check part of [Akka management](https://github.com/akka/akka-management/issues/364)
+`akka-cluster-http-management` includes an Akka Management route provider for a readiness and a liveness check. To use it
+add the following dependency:
```

Comment:
I wonder if this shouldn't be automatic, since bootstrap will not work without the management routes anyway.

Reply:
But that's for a different PR

Reply:
It'll work if you don't add the health checks to your deployment but ofc will result in traffic before the cluster is formed

Reply:
But perhaps a good idea anyway